src/cameras/detection/cam_detection.py

```diff
@@ -15,27 +15,24 @@ def find_available_cameras(self) -> FoundCamerasResponse:
         cv2_backend = self._determine_backend()
 
         cams_to_use_list = []
-        for cam_id in range(CAM_CHECK_NUM):
-            cap = cv2.VideoCapture(cam_id, cv2_backend)
+        for this_usb_port_number in range(CAM_CHECK_NUM):
+            cap = cv2.VideoCapture(this_usb_port_number, cv2_backend)
             success, image = cap.read()
 
             if success:
                 try:
-                    cams_to_use_list.append(
-                        RawCamera(
-                            webcam_id=cam_id,
-                        )
-                    )
+                    cams_to_use_list.append(this_usb_port_number)
                 finally:
                     cap.release()
 
         return FoundCamerasResponse(
-            camera_found_count=len(cams_to_use_list),
-            cams_to_use=cams_to_use_list,
+            number_of_cameras_found=len(cams_to_use_list),
+            list_of_usb_port_numbers_with_cameras_attached=cams_to_use_list,
             cv2_backend=cv2_backend,
         )
 
-    def _determine_backend(self):
+    @staticmethod
```

Comment:
I dont think notating this as a static method helps since the method is private

Reply:
makes sense, I'll remove that. 

PyCharm told me that it was a "static method" and I got excited 😅

Reply:
I see. Technically, since there is no reference to any `self` instance variables, it COULD be defined as static. thats the heuristic that Pycharm uses to recommend. Not always perfect :D